code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -688,6 +688,22 @@
 		M.bodytemperature = min(330, M.bodytemperature + (20 * TEMPERATURE_DAMAGE_COEFFICIENT)) //310 is the normal bodytemp. 310.055
 	return ..()
 
+/datum/reagent/consumable/ethanol/adminfreeze
+	name = "Admin Freeze"
+	id = "adminfreeze"
+	description = "Ultimate Punishment."
+	reagent_state = LIQUID
+	color = "#30F0FF" // rgb: 048, 240, 255
+	alcohol_perc = 0.5
+	drink_icon = "adminfreeze"
+	drink_name = "Admin Freeze"
+	drink_desc = "The ultimate punishment."
+	taste_description = "a series of bad decisions...BWOINK"
```

Comment:
```suggestion
	taste_description = "a series of bad decisions... BWOINK"
```
The smallest nitpick possible

Reply:
Doc and I actually debated even keeping the BWOINK in, since 1. It makes the sound when it's mixed, and 2. We really wanted it to make the sound when you drink it. I'm fine taking the BWOINK out altogether.